subsys/net/l2/openthread/Kconfig.thread

```diff
@@ -55,10 +55,14 @@ config OPENTHREAD_POLL_PERIOD
 
 config OPENTHREAD_MAX_CHILDREN
 	int "The maximum number of children"
-	range 10 512
+	range 10 511
 	default 32
 
 config OPENTHREAD_MAX_IP_ADDR_PER_CHILD
-	int "The maximum number of IPv6 address registrations per child."
+	int "The maximum number of IPv6 address registrations per child"
```

Comment:
Looks like these changes should be in some other PR, as the commit talks only about configuring platform info.

Reply:
I updated the commit description.

Reply:
Thanks. The changes look unrelated, could you split the commit to two, so this PR would have two commits for these changes.

Reply:
Done.